src/handlers/channel.ts

```diff
@@ -422,6 +421,49 @@ export async function editChannel(
   );
 }
 
+export async function editChannelOverwrite(
+  channelID: string,
+  overwrite: Overwrite,
+  reason?: string,
+) {
+  const hasManageChannelPerm = await botHasChannelPermissions(
+    channelID,
+    ["MANAGE_CHANNELS"],
+  );
+  if (!hasManageChannelPerm) {
+    throw new Error(Errors.MISSING_MANAGE_CHANNELS);
+  }
+
+  const { allow, deny, ...info } = overwrite;
```

Comment:
I'm not really sure how necessary this is. But whenever you can avoid making variables its a really good thing. Variables should only be done when you have to reuse them multiple times imo.  I only see these being used once.

In JS, any variable like this takes up space and until Garbage Collector runs this is wasted space. It's also quite performance heavy to create when we could just do `overwrite.allow` below.

Reply:
Removed that